tests/group_functions/group_scan.h

```diff
@@ -238,7 +238,7 @@ struct init_joint_scan_group {
 
       size_t work_group_size = work_group_range.size();
 
-      const size_t sizes[3] = {5, work_group_size / 2, 3 * work_group_size};
+      const size_t sizes[3] = {5, work_group_size / 2};
```

Comment:
Either declare 2-element array or explicitly initialize the 3rd element.

Why the second element is `work_group_size / 2`? Why can't we put 1 or 2 instead (+ check that HW is able to run the work of that size)?

Reply:
That means which should even more comment the code.